README.md

```diff
@@ -20,7 +20,7 @@ Install Ruby, e.g. using [RBenv](https://github.com/rbenv/rbenv):
 ```sh
 rbenv install `cat .ruby-version`
 bundle install
-pod install --verbose
```

Comment:
Let's leave the `--verbose` in though, it can be helpful when things don't work.